script.js

```diff
@@ -0,0 +1,133 @@
+  navItem.classList.remove('active-menu');
+  menuOpen = false;
+}
+
+navItemBtn.forEach((itemBtn) => {
+  itemBtn.addEventListener('click', () => {
+    closeMenu();
+  });
+});
+
+menuBtn.addEventListener('click', () => {
+  if (!menuOpen) {
+    openMenu();
+  } else {
+    closeMenu();
+  }
+});
+
+function homePage() {
+  const performers = document.querySelector('.performers');
+  const moreBtn = document.getElementById('more');
+  const arrow = document.getElementById('arrow');
+  let more = false;
+  const performersInfo = [
+    {
+      image: 'img/cho.jpg',
+      altImage: 'Seong-Jin Cho Picture',
+      name: 'Seong-Jin Cho',
+      performersTitles: `South Korean pianist, 
+        winner of first prize at the 17th Warsaw International Chopin 
+        Piano Competition.`,
+      performersResume: `He displayed an impressive 
+        variety of tonal colors and remarkable technique, dispatched 
+        with jaw-dropping panache. He had something to say and 
+        communicated it well.`,
+    },
+    {
+      image: 'img/Liu_Kate_square.jpg',
+      altImage: 'Kate Liu picture',
+      name: 'Kate Liu',
+      performersTitles: `American classical pianist born 
+        in Singapore.`,
+      performersResume: `Pianist Kate Liu gained international 
+      acclaim after winning the Bronze Medal and Best Mazurka 
+      Prize at the 17th International Fryderyk Chopin Competition 
+      in Warsaw, Poland.`,
+    },
+    {
+      image: 'img/640px-Avdeevea.jpg',
+      altImage: 'Yulianna Avdeeva picture',
+      name: 'Yulianna Avdeeva',
+      performersTitles: `Yulianna Andreevna Avdeeva is 
+        a Russian concert pianist.`,
+      performersResume: `
+        She became famous by 
+        winning first prize at the 16th edition of the Frédéric Chopin 
+        International Piano Competition, the first woman to win it 
+        since Martha Argerich, in 1965.`,
+    },
+    {
+      image: 'img/rafal-blechacz.jpg',
+      altImage: 'Rafał Blechacz picture',
+      name: 'Rafał Blechacz',
+      performersTitles: `Polish classical pianist 
+        who rose to fame after decisively winning the XV International 
+        Chopin Piano Competition in 2005.`,
+      performersResume: `
+        Rafał Blechacz is a Polish pianist, known for his
+         interpretations of the works of Frédéric Chopin.`,
+    },
+  ];
+  function diplayPerformers(num) {
+    performers.innerHTML = '';
+    for (let i = 0; i < (performersInfo.length - num); i += 1) {
+      const performersPresentation = `
+        <li class="d-flex align-items-center gap-3">
+        <img src="${performersInfo[i].image}" alt="${performersInfo[i].altImage}">
+        <section>
+          <h3 class="name">${performersInfo[i].name}</h3>
+          <article class="performers-titles">${performersInfo[i].performersTitles}</article>
+          <div class="light-bar"></div>
+          <p>${performersInfo[i].performersResume}</p>
+        </section>
+        </li>
```

Comment:
I kindly suggest you use the "ARTICLE" semantic tag (instead of UL/LI), this is ideal for repetitive content like this

Reply:
I understand your point of view, but only for this case, I thought using ul/li was okay, because on the page that is a list of the performers for the event.